### ISSUE TITLE
zoom not possible on all mobile devices

### ISSUE THREAD

**Issue by oparoz** [nextcloud/gallery#66]
_From @Flexmaen on August 4, 2015 21:52_

The gallery is doing a great job on mobile devices too (tested with Blackberry and Android Browser).

However, if you want to zoom a picture, this is only possible if you download it to your local folder and open it from there. I think it would be great if you also could zoom pictures somehow.

_Copied from original issue: owncloud/gallery#263_

**Comment by oparoz** [comment]
I agree. Unfortunately, the BigShot library we're using only captures iOS events.

That one of the reason I'm considering [changing Slideshow engine](https://github.com/owncloud/gallery/issues/158) or maybe offering a choice

**Comment by oparoz** [comment]
I've had a look at the Bigshot code and for it to work, someone would have to write new code converting the pinch distance into a scale factor. Not too difficult, but I'm not sure it's worth it given the fact that the Bigshot project has silently died.

**Comment by TroubleMaker2** [comment]
Even, I am looking for the zoom options in owncloud gallery or galleryplus apps. Please provide it.

**Comment by TroubleMaker2** [comment]
*Zoom option on mobile devices

**Comment by Huster-at-github** [comment]
Here too! - For me this is a show stopper for a gallery app.

**Comment by Gatak** [comment]
Why not simply remove the CSS code that prevent zooming?

**Comment by civita** [comment]
I am also looking for the zooming in the Android devices.
